OracleUnifiedDirectory/.gitignore

```diff
@@ -0,0 +1,16 @@
+*~
+OUD
+*.tmp
+*.tar
+*.tar.gz
+*.tgz
+*.out
+tmp*
+pipeline.yaml
+kubernetes
+samples/oud-add-dir-to-remote.env
+samples/stress-oud-dir.env
+imagetool/12.2.1.4.0/manifest*.properties
+CONTRIBUTING.md
```

Comment:
You have `CONTRIBUTING.md` in your `.gitignore` but it appears in your pull request. Not sure why. I recommend just `git rm` the file completely.